### ISSUE TITLE
[question] road to dawn's door hp issue

### ISSUE THREAD

**Issue by ywgATustcbbs** [ReactiveDrop/reactivedrop_public_src#361]
the door wont take damage and break any more when hit by many nades. is this a bug or some other mechanism? 

https://user-images.githubusercontent.com/11514506/196168348-713b8dff-9a58-4ab3-a2e3-6cc6fd4cd4e1.mp4

**Comment by BenLubar** [comment]
This is the door flip mechanic, where there's a damage buffer between a door changing which side it's dented on and falling.

**Comment by ywgATustcbbs** [comment]
> This is the door flip mechanic, where there's a damage buffer between a door changing which side it's dented on and falling.

I know this mechanism, But when looking at ～15s in slow motion, the door flipped to the up, Then the second nade came rapidly, hoever the door didn't fall. It is the same for  one or two seconds later. This is why i felt it weird

**Comment by BenLubar** [comment]
@GmaOrange was this fixed by your recent door change?

**Comment by GmaOrange** [comment]
Nope, this is different thing. But i personally like this alien vs marine nade mechanics
